yoyodyne/train.py

```diff
@@ -22,22 +22,29 @@ class Error(Exception):
     pass
 
 
-def _get_logger(experiment: str, model_dir: str, wandb: bool) -> List:
+def _get_logger(experiment: str, model_dir: str, log_wandb: bool) -> List:
     """Creates the logger(s).
 
     Args:
         experiment (str).
         model_dir (str).
-        wandb (bool).
+        log_wandb (bool).
 
     Returns:
         List: logger.
     """
     trainer_logger = [loggers.CSVLogger(model_dir, name=experiment)]
-    if wandb:
+    if log_wandb:
         trainer_logger.append(
             loggers.WandbLogger(project=experiment, log_model="all")
         )
+        # Tells PTL to log best validation acc
+        import wandb
+
+        wandb.define_metric("val_accuracy", summary="max")
```

Comment:
Maybe add a comment here about what you're trying to do and what the issue is. Something like:

```python
# This saves the max validation accuracy on this run, duplicating effort made by the PTL trainer.
# TODO: factor out this duplication.